src/utils/helpers/style-helper/style-helper.js

```diff
@@ -1,6 +1,10 @@
-import { classicTheme } from '../../../style/themes';
+import { aegeanTheme, classicTheme, mintTheme } from '../../../style/themes';
 
 // eslint-disable-next-line import/prefer-default-export
 export function isClassic({ name }) {
   return name === classicTheme.name;
 }
+
+export function isDLS({ name }) {
```

Comment:
I think this function is unnecessary. 
You could instead do !isClassic to achieve the same result.
This is also more in line with the fact, that there is only one classic theme, but many DLS themes and potential or more DLS themes coming in future.

Reply:
> I think this function is unnecessary.
> You could instead do !isClassic to achieve the same result.
> This is also more in line with the fact, that there is only one classic theme, but many DLS themes and potential or more DLS themes coming in future.

I had a doubt about this one because I didn't know what the theming policy was going to be. 

If we are supporting Classic and DLS variants only there is no need to go that defensive, makes much more sense to simplify using the negative.

Reply:
We are slowly removing the `classic` theme from the code base so I think `isClassic` will eventually be removed as well. IMO having `isDLS` has value as we may end up allowing greater theming flexibility in the future